index.html

```diff
@@ -8,26 +8,65 @@
   <title>Document</title>
 </head>
 <body>
-  <main>
-    <h1 class="title">
-      Awesome books
-    </h1>
+  <div class="main-container">
+    <header id="header">
+      <nav>
+        <ul class="header__ul">
+          <li><a class="header__logo" href="#">Awesome Books</a></li>
+          <li class="desktop-menu-container">
+            <ul class="desktop-menu">
+              <li id='showList'><a href="#">List</a></li>
+              <li id='showAddBook'><a class="middle-item" href="#">Add new</a></li>
+              <li id='showContact'><a href="#">Contact</a></li>
+            </ul>
```

Comment:
- [ ] To enhance the user experience of this design kindly ensure that the active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/148324245-8894c66c-eaa8-4672-a7b0-6454c5a9e370.png)



**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)